spec/service_spec.rb

```diff
@@ -36,6 +36,11 @@
     expect(service.memory).to eq(1024)
   end
 
+  it 'memory takes floats too' do
```

Comment:
Ready to merge, but test descriptions should be readable as a sentence. Something it `it 'allows floats for memory settings'`.